### PR TITLE
Bpmn coverage

### LINKED ISSUE #135: Restore visual representation of covered BPMN elements
The legacy Zeebe documentation contained a visual representation of the BPMN elements covered by Zeebe:

image

During documentation migration this was difficult to carry over and was replaced by a plain bullet list.

A PR #46 was prepared to restore the visual representation. 

As far as I know, @saig0 had some reservations about the proposed solution.

### PR DESCRIPTION
**Pull request by saig0**
* rework the BPMN coverage page
* replace the list of supported elements with a complete overview of BPMN elements
* mark all supported BPMN elements
* link the supported BPMN elements to their documentation pages
* apply the changes for all versions: latest, 1.0, 0.26, 0.25

was inspired by #46

closes #135

### CODE REVIEW

File: versioned_docs/version-1.0/reference/bpmn-processes/bpmn-coverage.md [hunk 1]
```diff
@@ -3,970 +3,690 @@ id: bpmn-coverage
...
 
-The following BPMN elements are currently implemented in Zeebe:
+export const Highlight = ({children, color}) => (
+<span style={{ backgroundColor: color, borderRadius: '5px', color: '#fff', padding: '0.2rem', }}>
+{children}
+</span>
+);
+
+The following BPMN elements in <Highlight color="#df8a13">orange</Highlight> are supported. Click on
+an element to navigate to the documentation.
```
Comment (saig0) [inline versioned_docs/version-1.0/reference/bpmn-processes/bpmn-coverage.md:13]: This part is copied from the example on the documentation: https://docusaurus.io/docs/markdown-features/react#using-jsx-in-markdown

File: versioned_docs/version-1.0/reference/bpmn-processes/bpmn-coverage.md [hunk 2]
```diff
@@ -3,970 +3,690 @@ id: bpmn-coverage
...
+import LaneSvg from './assets/bpmn-symbols/lane.svg';
+
+<div className="bpmn-symbol-container">
+    <a href="#">
+        <PoolSvg className="implemented" />
+    </a>    
+    <a href="#">
+        <LaneSvg />
+    </a>
+</div>
```
Comment (saig0) [inline versioned_docs/version-1.0/reference/bpmn-processes/bpmn-coverage.md:27]: All SVGs are imported as "themed" images. Following the description on the docs: https://docusaurus.io/docs/markdown-features/assets#inline-svgs

**Comment by saig0** [comment]
# Before
image

image

# After
image

image

image

image

**Review by menski — APPROVED** [review]
Thank you a lot this is great. One proposal we can also make annotation and group as supported as the user can already model them in the Camunda Modeler, wdyt?

**Comment by saig0** [comment]
> One proposal we can also make annotation and group as supported as the user can already model them in the Camunda Modeler, wdyt?

True. I could mark them as supported. In the platform docs, it is not marked as supported https://docs.camunda.org/manual/7.15/reference/bpmn20/

**Pull request merged**